spec/requests/posts_request_spec.rb

```diff
@@ -13,13 +13,9 @@
   end
 
   describe 'GET #show' do
-    before(:example) { get('/users/3/posts/2') }
+    before(:example) { get('/users/') }
     it 'Is a sucess' do
       expect(response).to have_http_status(:ok)
     end
```

Comment:
- [ ] Let's add test to ensures that the response body includes the correct placeholder text. For example. " Welcome to the index page"
Hint: `response.body`

Reply:
Hi @uwadonat,
Thanks for the review but  I think since the status is "ok" that means the body has been rendered.
I will try to learn more about how to use the `response.body` but now I'm out of time because of the deadline :smile:

Reply:
- [ ] As suggested by the previous reviewer and this is the requirement as well kindly add the test for checking the response body as well. I'm giving you a hint so that you can take reference.

`expect(response.body).to include("Welcome to user index page")` the text which I'm checking here depends upon what you have included in the post-show page.